blivet/osinstall.py

```diff
@@ -250,7 +250,7 @@ def _find_existing_installations(devicetree):
 
         if not os.access(get_sysroot() + "/etc/fstab", os.R_OK):
             util.umount(mountpoint=get_sysroot())
-            device.teardown(recursive=True)
+            device.teardown()
```

Comment:
Isn't this to prevent issues with the whole stack being activated in the ``device.setup()`` call above and the leaf ``device`` being teared down in the end leaving the rest of the stack active? Or is that no longer an issue? It definitely used to be.